### PR TITLE
DOC: Refer to NumPy, not pandas, in main page

### PR DESCRIPTION
**Pull request by dlax**

### CODE REVIEW

**Comment by bashtage** [comment]
If you want another one, you could correct the capitalization of pandas so that Pandas -> pandas.

File: doc/source/index.rst [hunk 1]
```diff
@@ -58,7 +58,7 @@ basic statistical operations, random simulation and much more.
     ^^^^^^^^^^
```
Comment (charris) [inline doc/source/index.rst:58]: The `^` bit looks misaligned.

Comment (charris) [reply doc/source/index.rst:58]: Curiously, it look good locally. Soft tab maybe?

The "Contributor's Guide" section title also looks incorrectly marked up with `^`.

**Comment by charris** [comment]
Thanks @dlax .

**Pull request merged**